openedx/features/discounts/models.py

```diff
@@ -0,0 +1,35 @@
+"""
+DiscountRestrictionConfig Models
+"""
+
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+from django.utils.encoding import python_2_unicode_compatible
+
+from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
+
+
+@python_2_unicode_compatible
+class DiscountRestrictionConfig(StackedConfigurationModel):
+    """
+    A ConfigurationModel used to manage restrictons for lms-controlled discounts
+    """
+
+    STACKABLE_FIELDS = ('enabled',)
+
+    @classmethod
+    def enabled_for_course(cls, course):
```

Comment:
the requirements suggested to use a stacked config model as in FBE. This uses the same mechanism/model and works the same way. the way this would work is one toggle would enable the discount for edx.org. then other toggles would disable the discount for individual orgs/courses/course runs  

if it is disabled for an org/course enabled_for_course will return false